airbyte-integrations/connectors/source-mailchimp/source_mailchimp/schemas/list_members.json

```diff
@@ -0,0 +1,407 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": ["object", "null"],
+    "title": "Members List",
+    "description": "Information about members in a specific list.",
+    "properties": {
```

Comment:
Can you set additional properties to true here? https://docs.airbyte.com/connector-development/schema-reference/